client/src/reducers/authReducer.js

```diff
@@ -1,36 +1,41 @@
 /**
  * This file exports our auth reducer that will handle all dispatched auth-related actions.
  */
-import { SIGN_IN_REQUEST, SIGN_IN_SUCCESS, SIGN_IN_FAIL, SIGN_OUT, REGISTER_SUCCESS } from '../actions/types';
+import { LOG_IN_REQUEST, LOG_IN_SUCCESS,LOG_IN_FAIL,LOG_OUT, REGISTER_SUCCESS, REMOVE_LOG_OUT_ALERT, DISPLAY_LOG_OUT_ALERT } from '../actions/types';
 import Cookies from 'js-cookie';
 
 const INITIAL_AUTH_STATE = { 
   signIn: Cookies.get('isLogin') ? true : false,
```

Comment:
Since you refactored naming from sign in/out -> log in/out, should we rename the state prop here for consistency?

Reply:
Thanks. Missed that one.